gammapy/data/pointing.py

```diff
@@ -335,38 +336,38 @@ def read(cls, filename, hdu="EVENTS"):
         Parameters
         ----------
         filename : str
-            File name
-        hdu : int or str
-            HDU number or name
+            Filename.
+        hdu : int or str, optional
+            HDU number or name. Default is "EVENTS".
 
         Returns
         -------
         pointing_info : `PointingInfo`
-            Pointing info object
+            Pointing info object.
         """
         filename = make_path(filename)
         header = fits.getheader(filename, extname=hdu)
         return cls.from_fits_header(header)
 
     @property
     def mode(self):
-        """See `PointingMode`, if not present, assume POINTING"""
+        """See `PointingMode`, if not present, assume POINTING."""
         return self._mode
 
     @property
     def fixed_altaz(self):
-        """The fixed coordinates in AltAz of the observation.
+        """The fixed coordinates of the observation in AltAz as a `~astropy.coordinates.SkyCoord` object.
```

Comment:
Should be consistent with how we write AltAz in the docstrings. Can you decide on one and implement it everywhere?

Reply:
`AltAz` is the name of the astropy coordinates class (`~astropy.coordinates.AltAz`). So I think that we should not go with this writing. I decide to go with `alt-az`, even though `alt-az` is an abbreviation for `altitude-azimuth` I think that everyone gets it.